app/views/exception_notifier/_data.text.erb

```diff
@@ -1,4 +1,4 @@
-<% job = @data[:sidekiq][:job] %>
+<% job = @data.dig(:sidekiq, :job) || {} %>
```

Comment:
I made this change because while I was working on the Ahoy changes, I caused an error at one point, but Exception Notification was not able to send the email with the exception because in this case there is no Sidekiq data present.